foodalert/static/foodalert/js/components/form-controller.vue

```diff
@@ -61,7 +61,7 @@
                 axios.post('/notification/', data, {"headers": headers})
                     .then(function(response) {
                         console.log(response);
-                        this.$router.push({ name: 'update'});
+                        this.$router.push({ name: 'permit'});
```

Comment:
Permit currently comes after submitting a form for a notification, this should come before. Also update our vue router